R/dummy.R

```diff
@@ -70,35 +70,35 @@
 #'
 #' @examples
 #' library(modeldata)
-#' data(okc)
-#' okc <- okc[complete.cases(okc), ]
+#' data(Sacramento)
+#' Sacramento <- Sacramento[complete.cases(Sacramento), ]
```

Comment:
No need to call `complete.cases()` here since `Sacramento` doesn't have any missing data